tests/mock_tikv_tests.rs

```diff
@@ -7,6 +7,7 @@ mod test {
     use tikv_client::{KvPair, RawClient};
 
     #[tokio::test]
+    #[ignore]
```

Comment:
Why ignore it?

Reply:
It's unstable. Sometimes it reports "deadline exceeded" error. And it doesn't provide much value for now.

Reply:
Ok